x-pack/plugins/stack_alerts/public/rule_types/geo_containment/readme.md

```diff
@@ -44,7 +44,7 @@ There are 3 separate tabs you'll need for a combination of loading and viewing t
 data. Since you'll be jumping between them, it might be easiest to just open them
 upfront. Each is preceded by `https://localhost:5601/<your dev env prefix>/app/`:
 - Stack Management > Data Views: `management/kibana/dataViews`
-- Stack Management > Alerts & Actions: `management/insightsAndAlerting/triggersActions/alerts`
+- Stack Management > Rules: `management/insightsAndAlerting/triggersActions/rules`
```

Comment:
Can you just delete this entire readme? Its out of date and replaced by https://www.elastic.co/guide/en/kibana/current/asset-tracking-tutorial.html.  If you don't want to do this in your PR that is OK and our team can put up a seperate PR to remove the file.

Reply:
@nreese Okay, deleted the file.